tests/difficulty_test.go

```diff
@@ -52,11 +52,8 @@ func TestDifficulty(t *testing.T) {
 
 	// files are 2 years old, contains strange values
 	dt.skipLoad("difficultyCustomHomestead\\.json")
-	dt.skipLoad("difficultyMorden\\.json")
-	dt.skipLoad("difficultyOlimpic\\.json")
 
 	dt.config("Ropsten", *params.RopstenChainConfig)
-	dt.config("Morden", *params.RopstenChainConfig)
```

Comment:
You can't skip-load these, if they still exist in the ehtereum/tests subrepo from which we load the tests. Are they files gone?

Reply:
https://github.com/ethereum/tests/blob/develop/BasicTests/difficultyMorden.json